config/locales/forms/2025/sales/sale_information.en.yml

```diff
@@ -165,6 +193,16 @@ en:
             hint_text: "Amount paid before any charges"
             question_text: "What is the basic monthly rent?"
 
+          mrent_staircasing:
+            prestaircasing:
+              check_answer_label: "Monthly rent prior to staircasing"
+              hint_text: "Amount paid before any charges"
+              question_text: "What was the basic monthly rent prior to staircasing?"
+            poststaircasing:
+              check_answer_label: "Monthly rent after staircasing"
+              hint_text: "Amount paid before any charges"
+              question_text: "What is the basic monthly rent after staircasing?"
```

Comment:
Once this branch is merged with the staircasing initial purchase branch, it might be worth refactoring this under the mrent key.